language/move-vm/types/src/gas_schedule.rs

```diff
@@ -289,3 +190,23 @@ pub fn calculate_intrinsic_gas(
         min_transaction_fee.unitary_cast()
     }
 }
+
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[repr(u8)]
+pub enum NativeCostIndex {
+    SHA2_256 = 0,
+    SHA3_256 = 1,
+    ED25519_VERIFY = 2,
+    ED25519_THRESHOLD_VERIFY = 3,
+    LCS_TO_BYTES = 4,
+    LENGTH = 5,
+    EMPTY = 6,
+    BORROW = 7,
+    BORROW_MUT = 8,
+    PUSH_BACK = 9,
+    POP_BACK = 10,
+    DESTROY_EMPTY = 11,
+    SWAP = 12,
+    SAVE_ACCOUNT = 13,
+}
```

Comment:
Should this go with the natives? Or is it better in with the gas?

Reply:
> Should this go with the natives? Or is it better in with the gas?

very good question and I don't know. The hope is to put up another PR with a table style declaration (macros likely) that combines it all. I have not done that work yet but I feel this moves us in that direction.
It is a funny one though, no? gas or natives.... hmmmmm.....